lib/channel.js

```diff
@@ -84,6 +84,11 @@ function Channel(apiToken) {
 					});
 				}
 
+				if (!!data.switch_to_channel && data.switch_to_channel.toString().match(/^\d+$/)) {
```

Comment:
!!data.switch_to_channel seems a bit redundant. Just checking the var itself will return the same results since JS will auto do the same conversion